app/views/open_ai/show.js.erb

```diff
@@ -0,0 +1,12 @@
+(() => {
+  const element = document.querySelector('[data-role="ai-prompt"]')
+  const messageElement = element.querySelector('[data-role="ai-message"]')
+  const form = element.querySelector("form")
+
+  form.remove()
+  messageElement.innerHTML = `
+    <%= @message %>
+    <p class="text-dark text-small"><em>This is an experimental feature and the results may not be great. I'm still trying to get better at understanding the Workshop, just like you!</em></p>
```

Comment:
```suggestion
    <p class="text-dark text-small"><em>Like all generative AI, I get things wrong sometimes, so be careful when using my advice. I'm still trying to get better at understanding the Workshop, just like you!</em></p>
```
Might be worth reiterating what the tooltip says just in case people don't hover over the help icon!

Reply:
I like that, nice